Stack: Migrate to CSS gap (#1606)
seek-oss/braid-design-system: .changeset/tasty-weeks-search.md

---
'braid-design-system': minor
---

---
updated:
  - Stack
---

**Stack:** Widen `component` support

With `Stack` no longer adding intermidiary elements, the `component` prop can now accept a wider range of elements.

> The list is kept to a white list of relevant elements to the component that do not require other HTML attributes, keeping in mind that props are not blindly spread in Braid.

Reviewer: ```suggestion
Valid options are kept to a white list of elements relevant to `Stack` that do not require other HTML attributes, keeping in mind that props are not blindly spread in Braid.
```
